src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const caseDefault = 'An error because of an unknown type';
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
You can also make a destructuring to get all the needed props of action (type, extraData, keysToRemove)